redisvl/extensions/llmcache/schema.py

```diff
@@ -26,6 +26,8 @@ class CacheEntry(BaseModel):
     """Optional metadata stored on the cache entry"""
     filters: Optional[Dict[str, Any]] = Field(default=None)
     """Optional filter data stored on the cache entry for customizing retrieval"""
+    dtype: str
```

Comment:
This should be something that's passed to the serialization method `to_dict`. `CacheEntry` itself doesn't need to hold onto this state.

Reply:
Shouldn't dtype stay bound to wherever we have the vector, to avoid any potential conflicts between the byte string and dtype?

Reply:
I'd like to think that our extension classes are responsible for the write/read of the data.. and thus elements in the CacheEntry don't need to know how they were stored. we can chat about this!